docs/tutorials/transformation-and-normalization/transformations-with-dbt.md

```diff
@@ -138,30 +193,56 @@ dbt run --profiles-dir=$NORMALIZE_DIR --project-dir=$NORMALIZE_DIR --full-refres
 Example Output:
 
 ```text
-Running with dbt=0.18.1
-Installing https://github.com/fishtown-analytics/dbt-utils.git@0.6.2
-  Installed from revision 0.6.2
+Running with dbt=0.19.1
+Installing https://github.com/fishtown-analytics/dbt-utils.git@0.6.4
+  Installed from revision 0.6.4
+
+Running with dbt=0.19.1
+Found 4 models, 0 tests, 0 snapshots, 0 analyses, 364 macros, 0 operations, 0 seed files, 1 source, 0 exposures
+
+Concurrency: 32 threads (target='prod')
 
-Running with dbt=0.18.1
-Found 1 model, 0 tests, 0 snapshots, 0 analyses, 302 macros, 0 operations, 0 seed files, 1 source
+1 of 1 START table model quarantine.covid_epidemiology....................................................... [RUN]
+1 of 1 OK created table model quarantine.covid_epidemiology.................................................. [SELECT 35822 in 0.44s]
 
-15:37:54 | Concurrency: 32 threads (target='prod')
-15:37:54 | 
-15:37:55 | 1 of 1 START table model quarantine.covid_epidemiology....................................................... [RUN]
-15:37:55 | 1 of 1 OK created table model quarantine.covid_epidemiology.................................................. [SELECT 17911 in 0.30s]
-15:37:55 | 
-15:37:55 | Finished running 1 table model in 0.51s.
+Finished running 1 table model in 0.63s.
 
 Completed successfully
 
 Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
 ```
+### Hooking up a custom dbt project to Airbyte syncs
```

Comment:
this feels out of place. I'm also slightly lost when reading this without some UI reference. this feels like it should either be a separate tutorial with pictures or part of the operations.md doc describing how to configure a private git repo.

Reply:
Ok i can make a part 3 tutorial then! (once I have a UI for pictures)